e2e/support/world.ts

```diff
@@ -0,0 +1,7 @@
+export function World() {
```

Comment:
Definitely! And more, World could be a class instead of a function, right? Gonna try this out..